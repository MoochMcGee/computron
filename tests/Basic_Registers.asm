[bits 16]

mov ax, 0x1122
mov bx, 0x3344
mov cx, 0x5566
mov dx, 0x7788
mov si, 0x99aa
mov di, 0xbbcc
mov bp, 0xddee
mov sp, 0xff00

db 0xf1
