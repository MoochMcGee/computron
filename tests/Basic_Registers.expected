(  config) Loading tmp.bin at 0x00010000
(  screen) No keymap to load!
(    init) BusMouse present
(    init) FDC present
(    init) IDE present
(    init) Keyboard present
(    init) PIC present
(    init) PIC present
(    init) PIT present
(    init) VGA present
(    init) VomCtl present
(    init) CMOS present
(    init) PS2 present
(     cpu) 0xF1: Secret shutdown command received!
(    dump) AX: 1122
(    dump) BX: 3344
(    dump) CX: 5566
(    dump) DX: 7788
(    dump) BP: DDEE
(    dump) SP: FF00
(    dump) SI: 99AA
(    dump) DI: BBCC
(    dump) CS: 1000
(    dump) DS: 0000
(    dump) ES: 0000
(    dump) SS: 0000
(    dump) FS: 0000
(    dump) GS: 0000
(    dump) CR0: 00000000
(    dump) A20: 0
(    dump) GDTR: {base=00000000, limit=0000}
(    dump) LDTR: {base=00000000, limit=0000}
(    dump) IDTR: {base=00000000, limit=0000}
(    dump) C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0
(    dump)   -  (B8 2211BB4433)
(    dump) 1000:00000018 F1             ??? 
